README.md

```diff
@@ -2,192 +2,71 @@
 
-- The trial statuses transitions map. A scheme which define the transition for a certain status to the next one.
+- The trial statuses transitions map. A scheme which defines the transition from a certain status to the next one.
 - Grace periods for each trial status.
-- The reward which will be transfer to the plaintiff as a compensation for the case of a valid claim.
-
-While the main SwearGame contract conducts a trial to resolve a specific case it iterates between different trial statuses.
-
-This TrialRules contract is aligned with TrialRulesAbstract.sol  and should implement the following ABIs:
-
-/// @notice getStatus - get next trial status according to witness state and the current trial state
-
-/// @param witnessStatus witness status (VALID , INVALID,PENDING)
-
-/// @param trialStatus current trial status
-
-/// @return status - next trial status - can be also GUILTY or NOT GUILTY.
-
-function getStatus(uint8 witnessStatus,uint8 trialStatus) returns (uint8 status);
-
-/// @notice getWitness - get witness according to the trial status
+- The reward which will be transferred to the plaintiff as a compensation for the case of a valid claim.
 
-/// @param trialStatus current trial status
+While the main Swear contract conducts a trial to resolve a specific case it iterates between different trial statuses.
 
-/// @return WitnessAbstract - return a witness contract instance
+This TrialRules contract is aligned with [TrialRulesAbstract](contracts/abstracts/trialrulesabstract.sol) contract.
 
-function getWitness(uint8 trialStatus) returns (WitnessAbstract);
-
-/// @notice getInitialStatus - get initial trial status
-
-/// @return status -
-
-function getInitialStatus() public returns (uint8 status);
-
-/// @notice expired - check expiration for a certain case and trial status.
-
-/// @return bool - true if expired otherwise false.
-
-function expired(bytes32 caseId,uint8 status) returns (bool);
-
-/// @notice startGracePeriod - start counting for a grace period for a certain case and status.
-
-/// @return bool - true if it actually start counting for the grace period
-
-///                false -if the grace period already started
-
-function startGracePeriod(bytes32 caseId,uint8 status) returns (bool);
-
-/// @notice getReward - return the reward for a valid case
-
-///
-
-/// @return reward - the reward for a valid case
-
-function getReward()returns (uint reward);
+### Trial registrar
 
+A contract which manages registrations and deposits for the game players.
+A player who wishes to register and/or deposit to the swear game should do that via this contract.
+The registrar contract is accessed by the main Swear contract and is aligned with  
+the ABIs defined at [RegistrarAbstract](contracts/abstracts/registrarabstract.sol) contract.
 
 
 ## Mirror game
 A very simple basic game where the service offers to mirror the client ENS("client.game") content on its own ENS("service.game").
 
 ### game flow
```

Comment:
Might want to clarify that this is an example game flow for a "reflection service"